src/nikhilMJun20/RemoveDuplicateStrings.java

```diff
@@ -0,0 +1,51 @@
+/*
+Assignment 36 : 31st July 2020
+
+Remove duplicate names from given String Array, print names in given format with its length. Input Order has to be maintained.
+String input[] = {"Aashtha","Kartikey","Piyush","Raj","Suparna","Piyush","Aashtha", "Barkha"}
+
+Output : Aashtha -> 7
+Kartikey -> 8
+Piyush -> 6
+Raj -> 3
+Suparna -> 7
+Barkha -> 6
+Total unique elements are :- 6
+
+Note: Use set interface.
+*/
+
+package nikhilMJun20;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class RemoveDuplicateStrings {
+	Set<String> getUniqueElements(String[] input) {
+		Set<String> uniqueSet = new LinkedHashSet<String>();
+		for(String name: input) {
+			uniqueSet.add(name);
```

Comment:
if you want to do it in a single method,
you can add one if condition,
if(uniqueSet.add(name)){
  /// print name with length
}